app.controllers/api.controllers.js

```diff
@@ -0,0 +1,17 @@
+
+const { fetchApi } = require("../app.models/api.models");
+
+
+
+exports.getApi = (req,res,next) => {
+  fetchApi().then((data) => {
+    
+    const parsed = JSON.parse(data)
+    console.log(parsed)
+     res.status(200).send(parsed);
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
Good work here, just be careful of variable names like `parsed` - what does this mean? what is the data? It's difficult for me to understand what the data is here without forcing me to go to the model function. To make this clearer it may be better to name `data` to `apiEndpointsData` and `parsed` to `parsedApiEndpoints` or something along those lines.

Good work adding the catch block also.